src/components/_Controllers/CMSLive.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import Routes from 'react-static-routes';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import ReactGA from 'react-ga';
+import { logPageView } from 'js/helpers/googleAnalytics';
+
+const CMSLive = ({ intl }) => {
+  ReactGA.initialize(process.env.GOOGLE_ANALYTICS, { titleCase: false });
+  logPageView();
+  return <Routes />;
+};
+
+CMSLive.propTypes = {};
```

Comment:
should we put `intl` in here?

Reply:
For HOC injection of a prop I didn't add it to propTypes since it's basically hard-coded as a prop, so it'll never not be there like other props that are fed in via data/react-static route json. And I hadn't done that for any other components with intl injected -- so just being consistent.

Reply:
word